### ISSUE TITLE
Italic ttf variant mislabelled as Medium

### ISSUE THREAD

**Issue by vaskark** [NerdyPepper/scientifica#45]
The italic ttf variant is mislabelled as Medium. Just notifying :)

Amazing font, btw. Keep up the great work.

**Comment by NerdyPepper** [comment]
No idea what causes this issue till date. The SFDs have to be edited manually each time to fix this, opening them in FF and saving them resets them the `medium`.

**Comment by Kozova1** [comment]
Is there any workaround for this?

**Comment by NerdyPepper** [comment]
@Kozova1 adding a step to change the font variant in the build script maybe?

**Comment by masaeedu** [comment]
@NerdyPepper Any chance you could make another release with the fixed label? The latest release appears to have this problem. Thanks!

**Comment by execb5** [comment]
Yeah, another release with the fixed label would be greatly appreciated. 👍 
Love this font!

**Comment by xukai92** [comment]
> No idea what causes this issue till date. The SFDs have to be edited manually each time to fix this, opening them in FF and saving them resets them the `medium`.

I'd like to resolve it manually to use it. Can you give a bit more info on this? For example what are SFDs and how do we edit them? Thanks!

**Comment by NerdyPepper** [comment]
@masaeedu @execb5 thank you for the kind words, the latest release should have the fix (apologies for the tardy reply).

@xukai92 SFD files are plain text file formats that are recognized by Font Forge, my initial idea was to modify the SFD using a text editor (find and replace all instances of 'Medium' with 'Italic'), but it seems Font Forge overwrites this attribute on save. The latest release, v2.2, however does something slightly different (I have set the corresponding OS2 attributes), this might persist the changes, I am not too hopeful however.

**Comment by NerdyPepper** [comment]
@xukai92 to answer your question:

 - edit the SFD with a text editor (there should be exactly two instances of 'Medium' with a capital M, replace those with Italic)
 - open the new SFD in Font Forge
 - export to OTB

**Comment by xukai92** [comment]
Thanks for the advice and the fix on the recent release. I confirm that this italic variant of ttf is not correctly labelled.
